slatedb/src/compaction_execute_bench.rs

```diff
@@ -260,11 +262,11 @@ impl CompactionExecuteBench {
 
     fn load_compaction_as_job(
         manifest: &StoredManifest,
-        job: &CompactorJob,
+        job: &Compaction,
```

Comment:
I kinda feel like we should do a pass to remove all misnamed variables and functions (job -> compaction, attempt -> compactionjobspec, etc). If you don't want to bother with that, I can take it as a follow-up.